recipes-core/packagegroups/packagefeed-ni-core.bb

```diff
@@ -24,6 +24,8 @@ RDEPENDS:${PN} = "\
 
 RDEPENDS:${PN}:append = "\
 	docker \
+	xorgxrdp \
```

Comment:
We are now building ARM from same codebase and I believe these packages are not applicable to ARM targets.

Can you move them into [packagegroup-ni-graphical](https://github.com/ni/meta-nilrt/blob/nilrt/master/scarthgap/recipes-core/packagegroups/packagegroup-ni-graphical.bb#L1) (I assume these are not xfce specific; if so they should go into [packagegroup-ni-xfce](https://github.com/ni/meta-nilrt/blob/nilrt/master/scarthgap/recipes-core/packagegroups/packagegroup-ni-xfce.bb#L1)).